docs/en/enterprise-edition/policy-reference/secrets-policies/secrets-policy-index/git-secrets-115.adoc

```diff
@@ -0,0 +1,49 @@
+== Azure Machine Learning web service API key detected in code
+
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+|TBD
+
+|Checkov Check ID
+|CKV_SECRET_115
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Frameworks
+|Git
+
+|===
+
+
+=== Description
+
+An Azure Machine Learning web service API key is essential for accessing and utilizing Azure Machine Learning web services. This key is critical for authenticating requests to the service and ensuring that only authorized applications can interact with it. Exposure of this key can lead to unauthorized access, misuse of the API, and potential data breaches. Securely managing this key is crucial to maintaining the security and integrity of your applications. Regularly reviewing and updating your security settings and keys is important to prevent unauthorized usage. For detailed security guidelines and best practices, refer to the latest https://learn.microsoft.com/en-us/purview/sit-defn-azure-machine-learning-web-service-api-key[Azure Machine Learning documentation and support] resources.
+
+=== Fix - Buildtime
+
+*Azure*
+
+If your Azure Machine Learning web service API key is exposed, it is crucial to take immediate action to revoke the compromised key and generate a new one to secure access to the service. Follow these steps to replace the exposed key and update all dependent applications:
```

Comment:
If your Azure Machine Learning web service API key is exposed, it’s crucial to take immediate action to prevent any unauthorized use. Regenerate a new key through your Azure account and replace the exposed key with the new key in all relevant applications. By regenerating a compromised Azure Machine Learning web service API Key and updating your application integrations, you not only secure your access to Azure Machine Learning web services but also maintain the confidentiality and integrity of your data flows.